cmd/meroxa/root/apps/run.go

```diff
@@ -21,17 +21,15 @@ import (
 	"fmt"
 
 	"github.com/meroxa/cli/cmd/meroxa/builder"
-	turbineCLI "github.com/meroxa/cli/cmd/meroxa/turbine_cli"
-	turbineGo "github.com/meroxa/cli/cmd/meroxa/turbine_cli/golang"
-	turbineJS "github.com/meroxa/cli/cmd/meroxa/turbine_cli/javascript"
-	turbinepy "github.com/meroxa/cli/cmd/meroxa/turbine_cli/python"
+	turbineCLI "github.com/meroxa/cli/cmd/meroxa/turbine"
+	turbineGo "github.com/meroxa/cli/cmd/meroxa/turbine/golang"
+	turbineJS "github.com/meroxa/cli/cmd/meroxa/turbine/javascript"
+	turbinepy "github.com/meroxa/cli/cmd/meroxa/turbine/python"
 	"github.com/meroxa/cli/log"
 )
 
 type Run struct {
 	flags struct {
-		// `--lang` is not required unless language is not specified via app.json
-		Lang string `long:"lang" short:"l" usage:"language to use (go | js)"`
```

Comment:
i just went ahead and did it because the flag seemed unused, which i based on the usage text not even referencing python 4 months after launch.  i can keep it as a separate commit so that appears in the release, unless you think there's more to discuss here in which case i can make it a separate PR.

Reply:
@janelletavares let's no worry about it now, but I would like us to consider tackling these type of changes in separate releases whenever possible.